README.md

```diff
@@ -122,6 +121,27 @@ If you don't want to display comments for a particular post you can disable them
 
 --
 
+### Social networks
+
+You can add links to the accounts you have on other sites, with respective icon, by adding one or more of the following options in your config:
+
+```yaml
+twitter_username: jekyllrb
+github_username:  jekyll
+dribbble_username: dribbble
+facebook_username: fb
+flickr_username: flickr
+instagram_username: ig
+linkedin_username: linkedin
+pinterest_username: pinterest
+stackoverflow_username: stackoverflow
```

Comment:
I'm not overly concerned about it. Just mentioned it as an afterthought.. :smiley: